src/ddmd/dmodule.d

```diff
@@ -319,7 +319,7 @@ extern (C++) final class Module : Package
     int isDocFile;              // if it is a documentation input file, not D source
     bool isPackageFile;         // if it is a package.d
     int needmoduleinfo;
-
+    uint unitTestCounter;       // how many unittests have been seen so far
```

Comment:
Document that unit test counter is per module, rather than global, so that it is reproducible regardless of how it is compiled.

Reply:
Done.